ballerina-test-automation/gradle.properties

```diff
@@ -1,7 +1,7 @@
-swan-lake-latest-version=swan-lake-2201.4.0
+swan-lake-latest-version=swan-lake-2201.3.0
```

Comment:
Should we change this version since master will be using for update 4 releases?

Reply:
@keizer619 WDYT?

Reply:
We should keep 2201.4.0